src/cryptography/sign.js

```diff
@@ -172,12 +172,19 @@ export const signAndPrintMessage = (message, passphrase, secondPassphrase) => {
 	return printSignedMessage(signedMessage);
 };
 
-export const signData = (data, passphrase) => {
+export const signDataWithPassphrase = (data, passphrase) => {
 	const { privateKey } = getPrivateAndPublicKeyBytesFromPassphrase(passphrase);
 	const signature = naclInstance.crypto_sign_detached(data, privateKey);
```

Comment:
`signDataWithPassphrase` should use `signDataWithPrivateKey` as it' implementation to avoid duplicating all the logic that comes after the `privateKey` is known, e.g.

```
export const signDataWithPassphrase = (data, passphrase) => {
 	const { privateKey } = getPrivateAndPublicKeyBytesFromPassphrase(passphrase);
        return signDataWithPrivateKey(data, privateKey);
}
```

Reply:
@webmaster128 yes, make sense.